microsite/src/main/tut/index.md

```diff
@@ -26,13 +29,15 @@ println(s"""libraryDependencies += "org.scalaz" %% "scalaz-zio" % "${scalaz.zio.
 
 # Why IO?
 
-Effect monads like `IO` are how purely functional programs interact with the real world. Functional programmers use them to build complex, real world software without giving up the equational reasoning, composability, and type safety afforded by purely functional programming.
+At the core of ZIO is `IO`, a powerful data type inspired by Haskell's `IO` monad. The `IO` data type allows you to model asynchronous, concurrent, and effectful computations as a pure value.
+
+Effect types like `IO` are how purely functional programs interact with the real world. Functional programmers use them to build complex, real world software without giving up the equational reasoning, composability, and type safety afforded by purely functional programming.
 
 However, there are many practical reasons to build your programs using `IO`, including all of the following:
 
  * **Asynchronicity**. Like Scala's own `Future`, `IO` lets you easily write asynchronous code without blocking or callbacks. Compared to `Future`, `IO` has significantly better performance and cleaner, more expressive, and more composable semantics.
- * **Composability**. Purely functional code can't be combined with impure code that has side-effects without the straightforward reasoning properties of functional programming. `IO` lets you wrap up all effects into a purely functional package that lets you build composable real world programs.
- * **Concurrency**. `IO` has all the concurrency features of `Future`, and more, based on a clean fiber concurrency model designed to scale well past the limits of native threads. Unlike other effect monads, `IO`'s concurrency primitives do not leak threads.
+ * **Composability**. Purely functional code can't be combined with impure code that has side-effects without sacrificing the straightforward reasoning properties of functional programming. `IO` lets you wrap up all effects into a purely functional package that lets you build composable real world programs.
+ * **Concurrency**. `IO` has all the concurrency features of `Future`, and more, based on a clean fiber concurrency model designed to scale well past the limits of native threads. Unlike other effect monads, `IO`'s concurrency primitives do not leak resources.
```

Comment:
> Unlike other effect monads

There may be a more positive way to phrase that (except if you intend to use a competitive tone in the copywriting).

Reply:
I agree. I will pull out this whole section, rewrite it (it’s very old), and put it in a new page about IO. Doesn’t belong here but didn’t want to do too much in one PR.